setup.py

```diff
@@ -570,26 +570,46 @@ def determine_gl_flags():
 
 def determine_sdl2():
     flags = {}
+    # This is redundant; the only time determine_sdl2 is called is preceded by
+    # verification that this condition is already satisfied
     if not c_options['use_sdl2']:
         return flags
 
+    # Check the environment for a Kivy SDL2 path (all platforms) or for Include
+    # and Lib (intended to catch Windows installation, see #4941)
     sdl2_path = environ.get('KIVY_SDL2_PATH', None)
 
-    if sdl2_flags and not sdl2_path and platform == 'darwin':
-        return sdl2_flags
+    # SDL2 path was not contained in the environment; attempt to infer it
+    if not sdl2_path:
+        if platform == 'darwin' and sdl2_flags:
+            return sdl2_flags
+        # no pkgconfig info, or we want to use a specific sdl2 path, so perform
+        # manual configuration
+
+        # Not on OSX, and/or no sdl2_flags defined
+        sdl2_paths = []
+        sdl2_path_candidates = []
+        # Check in the executable's include dir
+        sdl2_path_candidates.append(
+            join(dirname(sys.executable), 'include', 'SDL2'))
+        # For every path in the %INCLUDE% (Windows), add the SDL2 subdir
+        sdl2_path_candidates.extend(
+            [join(p, 'SDL2') for p in
+             environ.get('INCLUDE', '').split(os.pathsep)]
```

Comment:
I was never using the build tools, I was using MSCV, and following the pre-existing compile directions exactly. Where is build tools coming from? From the comment in the docs? That was there before this PR; I had nothing to do with it.

Venv and system python have different directory structures; that's the root cause of this whole problem. Because the install directions (used to, and in this PR, again do) specify that the %INCLUDE% has to be set, I made the decision to (more reliably) pull the SDL location from it than from inferring directory structure of the venv, in the hopes of a more robust solution.